README.md

```diff
@@ -20,11 +20,16 @@ with building complex applications over the years.
 - Install rails `7.0.1`
 - Install node `16.13.2` (For creating web application)
 
+If running on Apple M1:
+
+- Set bundle platform to ruby `bundle config set --global force_ruby_platform true`
+- To build docker image, please make sure to set platform to AMD64 `export DOCKER_DEFAULT_PLATFORM=linux/amd64`
```

Comment:
Do you have a reference/source for this workaround? We might need to have it on other templates too.

Reply:
I have issue when building the image, Chrome is failed to installed

> #9 7.669 E: Unable to locate package google-chrome-stable
![Screen Shot 2565-03-17 at 16 05 22](https://user-images.githubusercontent.com/6965195/158776190-1fbace6d-314f-4082-bdaa-7e19af5b44bc.png)

and seems it is because we need to supply `--platform` when doing docker build

https://medium.com/no-sliver-bullet/apple-m1-%E8%AE%93%E4%BD%A0%E7%9A%84-docker-build-%E4%B8%8D%E8%B5%B7%E4%BE%86%E4%BA%86%E5%97%8E-bbe133d6d1bb

From docker documentation:
- https://docs.docker.com/engine/reference/commandline/build/#options
- https://docs.docker.com/desktop/mac/apple-silicon/#known-issues
- There are also https://docs.docker.com/desktop/multi-arch/ But I still haven't played much on it. Still don't know about it much.

We can supply that option in many places, `--platform` when build through cli, `platform` in docker compose file or through ENV `export DOCKER_DEFAULT_PLATFORM=linux/amd64`

Now I just set that ENV in my zshrc 👍

Reply:
For `bundle config set --global force_ruby_platform true`, without it, the `Gemfile.lock` will generate

 ```yaml
 PLATFORMS
  arm64-darwin-21
 ```
 (While usually we have platform ruby)
 
 The project is created successfully but when running docker container, it fails (I think maybe because platform is different?). The error is from Nogokiri
 
https://nokogiri.org/tutorials/installing_nokogiri.html#how-can-i-avoid-using-a-precompiled-native-gem

🛠️  To fix: we can just add ruby to platforms
 
```yaml
PLATFORMS
  arm64-darwin-21
  ruby
```
 
I found `bundle config set --global force_ruby_platform true` is easier, so we do not have to keep editing the lock file. 👀  

But after I tried to reproduce, it does not happen anymore (maybe because we set platform 💭 ). Now `Gemfile.lock` in the container is automatically modified to

```yaml
PLATFORMS
  arm64-darwin-21
  x86_64-linux
```

I'll revisit this config again tomorrow 🤔

Reply:
I can confirm `bundle config set --global force_ruby_platform true` is not required anymore. Maybe it's from platform setting:

On local machine:
![Screen Shot 2565-03-18 at 10 41 55](https://user-images.githubusercontent.com/6965195/158935397-de9f0c13-3970-4fc4-abe0-2b656d7f4123.png)

Inside container, The additional platform is added: 
![Screen Shot 2565-03-18 at 10 55 46](https://user-images.githubusercontent.com/6965195/158935438-11dad74b-8e47-4554-8543-14f0e41d0f29.png)

I've updated the readme on [f502949](https://github.com/nimblehq/rails-templates/pull/320/commits/f5029494ef98ef3aa4ff52acf34c2e5e0af83eb3) 👍